chef_master/source/resource_file.rst

```diff
@@ -259,9 +259,11 @@ This resource has the following properties:
 ``verify``
    **Ruby Types:** String, Block
 
-   A block or a string that returns ``true`` or ``false``. A string, when ``true`` is executed as a system command.
+   Allows verification of a file's contents before it is created. Creates a temp file and then allows execution of commands or Ruby code to make sure the file meets the needed structure.
```

Comment:
"fits the needed structure" doesn't seem to quite capture the intention here, but I'm struggling to come up with better words.
"to ensure the new file is fit for purpose"? I dunno that that's any better.
"the new file is valid"?